docs/sriov.md

```diff
@@ -284,6 +284,16 @@ all this to work. More details on usage can be found in
 and [SR-IOV operator](https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md)
 user documentation.
 
+> **NOTE:**  When using openshift SRIOV operator,
+> The `NetworkAttachmentDefinition` will be  created automatically, 
+> based on the `SriovNetwrok` object. You could also not use the operator. 
```

Comment:
typo `SriovNetwrok`

The ` You could also not use the operator.` does not fit well here.
Do you want to say that with the Operator, the NAD is defined with VLAN=0 if no VLAN is specified?
Without the Operator, one needs to set it explicitly to 0, otherwise the SR-IOV DP will keep the existing VF VLAN untouched (which may be non 0).

Reply:
> typo `SriovNetwrok`

Thanks. Fixed.
> 
> The ` You could also not use the operator.` does not fit well here.
> Do you want to say that with the Operator, the NAD is defined with VLAN=0 if no VLAN is specified?
> Without the Operator, one needs to set it explicitly to 0, otherwise the SR-IOV DP will keep the existing VF VLAN untouched (which may be non 0).


I wanted to describe just what  effectively will happen. In this case, when not declaring VLAN,  what will effectively happen is either the VLAN will be cleared (in SriovNetwork) , or it will stay in tact (NetworkAttachmentDefenition).
I wanted to avoid describing what exactly the operator is doing to avoid confusion.
I did add another sentence about how to clear vlan id from the vf when using NAD I hope it is clearer now.
What do you think?